app/components/WalletProvider.tsx

```diff
@@ -34,19 +34,20 @@ export default function WalletProvider({ children, network }: { children: ReactE
       new MetaMaskConnector({ chains }),
       // TODO implement COINBASE WALLET
       new CoinbaseWalletConnector({
-        chains,
+        chains: chains,
         options: {
+          jsonRpcUrl: `http://127.0.0.1:8545/`,
```

Comment:
hmmm this would work for running a node locally, what about using infura or 3rd part provider does this break or get ignored?

Reply:
Overlooked cleaning up this file after trying everything to debug it. This and chains: chains are reverted in the next commit